package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "hedgedoc.github.io",
+  "version": "1.0.0",
+  "repository": "git@github.com:hedgedoc/hedgedoc.github.io",
```

Comment:
This is not a valid package.json field.
See https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

Reply:
This is what yarn added here :shrug:

Reply:
changed